tests/cmds/object_files/spends/spend.json

```diff
@@ -2,4 +2,4 @@
           "parent_coin_info": "0x0000000000000000000000000000000000000000000000000000000000000000",
           "puzzle_hash": "0x0000000000000000000000000000000000000000000000000000000000000000"},
           "puzzle_reveal": "0x01",
-          "solution": "ffff32ffb080df54b2a616f5c79baaed254134ae5dfc6e24e2d8e1165b251601ceb67b1886db50aacf946eb20f00adc303e7534dd0ff2480ffff32ffb080df54b2a616f5c79baaed254134ae5dfc6e24e2d8e1165b251601ceb67b1886db50aacf946eb20f00adc303e7534dd0ff248080"}
+          "solution": "0xffff32ffb080df54b2a616f5c79baaed254134ae5dfc6e24e2d8e1165b251601ceb67b1886db50aacf946eb20f00adc303e7534dd0ff2480ffff32ffb080df54b2a616f5c79baaed254134ae5dfc6e24e2d8e1165b251601ceb67b1886db50aacf946eb20f00adc303e7534dd0ff248080"}
```

Comment:
Curious if the need to make this change is accidental and constitutes a breaking change in main.  Something we should dive into if possible.

Reply:
it's from the switch to using `make_spend` which seems to require the '0x' which previously was optional. It's also more consistent now, using it for all the fields in the json